roles/metrics/tasks/start-prometheus-node-exporter.yml

```diff
@@ -0,0 +1,64 @@
+- name: Download node exporter from Github
```

Comment:
See other comment, there's a playbook in the devops repo that does the node-exporter setup already (using a role, no need for manual shell commands)

Reply:
Commented there already, the external role also does the same steps. Just a question of which approach we want to go with.